src/main/java/com/crowdin/cli/commands/actions/StringAddAction.java

```diff
@@ -49,71 +49,80 @@ class StringAddAction implements NewAction<ProjectProperties, ProjectClient> {
     @Override
     public void act(Outputter out, ProjectProperties pb, ProjectClient client) {
         CrowdinProjectFull project = ConsoleSpinner.execute(out, "message.spinner.fetching_project_info", "error.collect_project_info",
-            this.noProgress, false, client::downloadFullProject);
+                this.noProgress, false, client::downloadFullProject);
         boolean isStringsBasedProject = Objects.equals(project.getType(), Type.STRINGS_BASED);
         boolean isPluralString = one != null || two != null || few != null || many != null || zero != null;
 
         List<Long> labelIds = (labelNames != null && !labelNames.isEmpty()) ? this.prepareLabelIds(client) : null;
+        Branch branch = BranchUtils.getOrCreateBranch(out, branchName, client, project, false);
```

Comment:
For FB projects it doesn't make sense because it creates an empty branch with no files so it will fail anyway.

Let's keep this behavior for SB projects only.